tools/coverage/git/git.go

```diff
@@ -30,27 +29,17 @@ const (
 
 // hasGitAttr checks git attribute value exist for the file
 func hasGitAttr(attr string, fileName string) bool {
-	//fmt.Printf("filename=*%s*\n", fileName)
-	attrCmd := exec.Command("git", "check-attr", attr, "--", fileName)
-	valCmd := exec.Command("cut", "-d:", "-f", "3")
-
-	pr, pw := io.Pipe()
-	attrCmd.Stdout = pw
-	valCmd.Stdin = pr
 	var val bytes.Buffer
-	valCmd.Stdout = &val
+	attrCmd := exec.Command("git", "check-attr", attr, "--", fileName)
```

Comment:
It looks you can save a few lines by using `pkg/test/cmd`, like in https://github.com/knative/test-infra/blob/42c9b74cda686778f2eb215c461325bd84a8551e/tools/cleanup/cleanup.go#L94. But it's up to you.